core.js

```
@@ -611,7 +611,17 @@ class FileTypeParser {
 		}
 
 		if (this.checkString('%PDF')) {
-			await tokenizer.ignore(1350);
+			try {
+				await tokenizer.ignore(1350);
+			} catch (error) {
+				if (error instanceof strtok3.EndOfStreamError) {
+					return {
+						ext: 'pdf',
+						mime: 'application/pdf',
+					};
+				}
```

You are swallowing other errors.